### PR TITLE
Added the note to the readme about the official package.

### PR DESCRIPTION
**Pull request by indualagarsamy**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,3 +1,5 @@
+**NOTE: This repository is no longer maintained. If you're looking for an officially supported MongoDB persistence for NServiceBus Versions 7 and above, please use [NServiceBus.Storage.MongoDB](https://github.com/particular/nservicebus.storage.mongodb). For documentation and samples, please refer to: https://docs.particular.net/persistence/mongodb/**
```
Comment (timbussmann) [inline README.md]:
```suggestion
**NOTE: This repository is no longer maintained. If you're looking for an officially supported MongoDB persistence, please use [NServiceBus.Storage.MongoDB](https://github.com/particular/nservicebus.storage.mongodb). For documentation and samples, refer to: https://docs.particular.net/persistence/mongodb/** 
```

**Review by HindyLambrozo — APPROVED** [review]

**Comment by HindyLambrozo** [comment]
can you give me write access so that my approval counts and i can include your suggestion @timbussmann

**Comment by timbussmann** [comment]
@HindyLambrozo: invited you to the labs org.

**Comment by timbussmann** [comment]
Closing this PR. The suggested changes have been merged via #28 as we couldn't merge the proposed changes to the fork.

**Pull request closed without merging**